driver/postgresql/driver.go

```diff
@@ -40,8 +41,16 @@ type Conn struct {
 
 // HasTable check if the schema has the migration table.
 func (c *Conn) HasTable() (bool, error) {
+	pgSchema := "public"
+	pgTable := c.table
+	if strings.Contains(pgTable, ".") {
+		parts := strings.Split(pgTable, ".")
+		pgSchema = parts[0]
+		pgTable = parts[1]
+	}
+
 	var name string
-	err := c.db.QueryRow(`SELECT table_name FROM information_schema.tables WHERE table_catalog = $1 AND table_name = $2`, c.schema, c.table).Scan(&name)
+	err := c.db.QueryRow(`SELECT table_name FROM information_schema.tables WHERE table_catalog = $1 AND table_name = $2 AND table_schema = $3`, c.schema, pgTable, pgSchema).Scan(&name)
```

Comment:
Nitpicking too, but I would prefer the columns to be ordered by importance: catalog, schema, table. Also, the schema variable should be renamed to `catalog` for this driver I guess?

Reply:
I'll do this.